### PR TITLE
Fixed Appsflyer UserId not sent, Screen & Page naming

### PR DESCRIPTION
**Pull request by desusai7**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: v0/destinations/af/transform.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ function responseBuilderSimple(payload, message, destination) {
   const updatedPayload = {
     ...payload,
     eventTime: message.timestamp,
-    customer_user_id: message.user_id,
+    customer_user_id: message.userId,
```
Comment (saikumarrs) [inline v0/destinations/af/transform.js:59]: Should support both for backward compatibility?

File: v0/destinations/af/transform.js [hunk 2]
```diff
@@ -207,12 +207,17 @@ function processSingleMessage(message, destination) {
       break;
     }
     case EventType.SCREEN: {
-      const eventName = EventType.SCREEN;
+      const eventName = `Viewed ${message.name ||
+        message.event ||
+        get(message, "properties.category") ||
+        ""} Screen`;
```
Comment (saikumarrs) [inline v0/destinations/af/transform.js:213]: Updating event name might impact existing customers

File: v0/destinations/af/transform.js (hunk 1 above)
Comment (desusai7) [reply v0/destinations/af/transform.js]: Actually the key `user_id` is obsolete now. So no events will come with that key. So ignoring this as we discussed

File: v0/destinations/af/transform.js (hunk 2 above)
Comment (desusai7) [reply v0/destinations/af/transform.js]: Agree with you, so provided a switch on the dashboard for the user to enable the screen/page name in the screen/page call.

**Review by saikumarrs — APPROVED** [review]

**Review by ruchiramoitra — DISMISSED** [review]

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**